runbooks/workflow/payments/add_agency_dashboard_data_source.md

```diff
@@ -0,0 +1,185 @@
+- [Create a New Agency Dashboard and the Comprising Questions](#create-a-new-agency-dashboard-and-the-comprising-questions)
+
+## Add a New Agency Data Source to Metabase and Create Permissions
+
+As new agencies are introduced to the contactless payments program we will need to access their data within Metabase for use in their payments dashboard and other analysis. Because we use a [row access policy](https://github.com/cal-itp/data-infra/blob/main/warehouse/macros/create_row_access_policy.sql) in the warehouse code to limit access to data to authorized parties this is a multi-step process.
+
+### Create a new service account and row access policy
+
+1. To begin, create a new service account for the agency within the Google Cloud Platform console, which will be used to allow Metabase read-access to the agency's payments data.
+
+- Navigate to: <https://console.cloud.google.com/iam-admin/serviceaccounts> in the `cal-itp-data-infra` project
+- Select `+ Create Service Account` in the top-center of the page
+- Populate the `Service account ID` field using the convention: `[agency-name]-payments-user`, then select `Create and Continue`
+- Within the `Grant this service account access to the project` section, assign the role `Agency Payments Service Reader`
+- Select `Done`
+
+2. Download the service account key for the service account you've just created
+
+- After selecting `Done` in the previous section you'll be returned to the list of existing service accounts. Click into the service account that you just created.
+- Select `Keys` from the top-center of the page and then select the `Add Key` dropdown. Choose the `Create new key` selection within that.
+- Keep the default key type `JSON` and select `Create`
+- This will download a JSON copy of the service accout key to your local environment, which will be used in later steps within Metabase
+
+3. Open a new branch and edit the [create_row_access_policy macro](https://github.com/cal-itp/data-infra/blob/main/warehouse/macros/create_row_access_policy.sql)
+
+Duplicate an existing row access policy within the file and append to the bottom , before the `{% endmacro %}` text.
+
+The contents of the policy you're duplicating should look like this:
+
+```
+{{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = '[agency-name]',
+    principals = ['serviceAccount:`agency-name`-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }};
+```
+
+Substitute the following fields with the appropriate information for the agency that you are adding:
+
+- `filter_value` which is the Littlepay `participant_id` for the agency
+- `principals` which is the email address for the service account that was created in step #1
+
+Open a PR and merge these changes. If you'd like access to the results of this policy before the next time the `transform_warehouse` DAG is run, you will need to run it manually.
```

Comment:
Would specify that they should trigger the DAG with a selector (as described in [the README for the DAG task](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/transform_warehouse)); `{"dbt_select": "models/mart/payments"}` should work.